tests/bounties/solidity-bounty/Makefile

```diff
@@ -1,5 +1,5 @@
 ARCH=$(shell uname -m)
```

Comment:
@guidanoli Is this variable really needed? Sorry if it is a dummy question, but I am derusting my Makefile knowledge after maaaaaaaaany year.

Reply:
Yeah, I think this variable is not necessary here.